modules/users/client/views/authentication/signin.client.view.html

```diff
@@ -49,42 +49,40 @@
                   </span>
                 </div>
               </div>
-              <a ui-sref="forgot({ userhandle: auth.credentials.username })"
-                 class="btn btn-inverse"
-                 ng-class="{ 'btn-lg btn-block': auth.authError }"
-                 tabindex="4">
-                Forgot?
-                <span ng-if="auth.authError" class="hidden-xs">
-                  Recover your password
-                </span>
-              </a>
-              <div class="text-center form-group">
-                <br><br>
+              <div class="form-group">
                 <button type="submit"
-                        class="btn btn-default btn-action"
+                        class="btn btn-default btn-lg"
```

Comment:
I'd leave it as it was so Login button is bigger

Reply:
`btn-action` looks too big to my taste. It is also not a Bootstrap standard.

Reply:
>It is also not a Bootstrap standard.

Bootstrap is just a base framework and not a design system per-se, it's okay to have additional components. ;-)

`btn-action` is big indeed, it's for "call to action" type of interactions and not designed to be used in forms.

Reply:
OK, fair enough